cmd/flags.go

```diff
@@ -131,3 +131,21 @@ func roleFlag() cli.Flag {
 		Usage: "Specify a team role to be used",
 	}
 }
+
+func githubAuthFlags() []cli.Flag {
+	return []cli.Flag{
+		cli.BoolFlag{
+			Name:  "github",
+			Usage: "--github",
+		},
+		cli.BoolFlag{
+			Name:  "github-user",
+			Usage: "--github-user",
```

Comment:
Should we call this `basic` rather than user? since basic auth will be a more recognizable term?